tests/createWcsWithSip.py

```diff
@@ -71,7 +63,7 @@ def tearDown(self):
         import lsst.meas.astrom.astrometry_net as an
         an.finalize()
 
-    def testBigXy0(self):
+    def xtestBigXy0(self):
```

Comment:
Was disabling this test intentional?  If so, maybe use `@unittest.skip(msg)` instead?